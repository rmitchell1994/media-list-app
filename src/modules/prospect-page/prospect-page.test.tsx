import React from "react";
import { render } from "../../../test/test-utils";
import ProspectPage from "./prospect-page";

describe("Prospect Page", () => {
  it("should render component", () => {
    const { asFragment } = render(<ProspectPage />);

    expect(asFragment()).toMatchSnapshot();
  });
});
