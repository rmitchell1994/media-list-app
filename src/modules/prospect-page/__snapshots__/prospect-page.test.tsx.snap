// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Prospect Page should render component 1`] = `
<DocumentFragment>
  <div>
    Prospect Page
  </div>
</DocumentFragment>
`;
